.travis.yml

```diff
@@ -1,12 +1,17 @@
-sudo: false
-branches:
```

Comment:
Do we not need this anymore?  This used to be required to stop duplicate tests on subsequent pushes to a branch that a PR is already opened for.